app/js/DocArchiveManager.js

```diff
@@ -45,6 +47,25 @@ async function archiveAllDocs(projectId) {
   })
 }
 
+async function archiveSingleDoc(projectId, docId) {
```

Comment:
What about `archiveDocById`? -- I think this more clearly differentiates this from the method below.

Reply:
Done and force-pushed as https://github.com/overleaf/docstore/pull/84/commits/f116c61744fd40d47b1511ebd0029710343ba8f8